### ISSUE TITLE
Staging Review finding: `How to check in` link opens in a new tab without notice

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#90774]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Modernized Check-in
**Product:** Pre-Check-in
**Feature:** Medication Review

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @briandeconinck (Accessibility)

### Description

On the `/complete/` and `/appointment-details/` views, the "Find out how to check in" link unexpectedly opens in a new tab.

## Recommended action

Our [guidance for link behavior](https://design.va.gov/components/link/#behavior) says to only open in a new tab if user data/progress could be lost. The link should either not open in a new tab (since the form flow is complete) or, if there is a reason a user might lose their progress if they navigate to this link, the link text should include "(opens in a new tab)" in its link text.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** [Link ](https://design.va.gov/components/language-toggle)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** User flow

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by benbrasso-agile6** [comment]
@wullaski change the link text to:

Find out how to check in (opens in new tab)

**Comment by loripusey** [comment]
Update content for both of these /complete/ and /appointment-details/

**Comment by benbrasso-agile6** [comment]
LGTM

**Comment by brianseek** [comment]
thanks @benbrasso-agile6! closing
